traffic_ops/app/db/migrations/20191029_add_dns_challenges.sql

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
You need to change the name of this file to something more recent, like `20200204_...` in order for the migrations to work properly.